src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -816,6 +816,15 @@ public static unsafe SocketError GetAvailable(SafeCloseSocket handle, out int av
             return err == Interop.Error.SUCCESS ? SocketError.Success : GetSocketErrorForErrorCode(err);
         }
 
+        public static unsafe SocketError GetAtOutOfBandMark(SafeCloseSocket handle, out int available)
```

Comment:
"available" seems like the wrong name here (I think it's effectively a bool, right?).  Maybe just "result"?

Reply:
(and elsewhere)

Reply:
Fixed